sklearn/preprocessing/tests/test_data.py

```diff
@@ -937,6 +937,29 @@ def test_maxabs_scaler_large_negative_value():
     assert_array_almost_equal(X_trans, X_expected)
 
 
+def test_maxabs_scaler_transform_one_row_csr():
+    """Check MaxAbsScaler on transforming csr matrix with one row"""
+    X_train_csr = sparse.csr_matrix([[1., 0., 3.],
+                                     [0., 2., 4.]])
+
+    scaler = MaxAbsScaler()
+    X_train_trans =_csr = scaler.fit_transform(X_train_csr)
+    X_train_expected = sparse.csr_matrix([[1., 0., 0.75],
+                                          [0., 1., 1.]])
+    assert_array_almost_equal(X_train_trans.A, X_train_expected.A)
```

Comment:
please use `X_train_trans.toarray()` instead of `X_train_trans.A` to be more explicit / readable.